### ISSUE TITLE
Parcel build segfaults with newer lmdb version (exit code 129)

### ISSUE THREAD

**Issue by valkum** [parcel-bundler/parcel#7720]
# 🐛 bug report

The parcel build command segfaults node when run from yarn workspaces foreach.

Per https://github.com/parcel-bundler/parcel/issues/7645 this might be related to lmdb. Pinning lmdb to a lower version resolves the issue as far as I can see.

## 🎛 Configuration (.babelrc, package.json, cli command)

Run `parcel build` for each workspace package in serial but started using the parallel version of yarn workspace-tools

E.g. `yarn build:libs` in https://github.com/valkum/parcel-segfault-test
which runs `yarn workspaces foreach -vptR`

## 🤔 Expected Behavior

The command runs through without segfaulting producing the build library files.

## 😯 Current Behavior

With an empty parcel-cache in roughly 70% of cases `yarn workspaces` says one command (most likely the second workspace package) exited with status code 129. There is a coredump generated by linux (I can provide that if needed) for the node process that runs parcel build.

## 💁 Possible Solution

pin lmdb to 2.1.7

## 🔦 Context

We want to use parcel as an easy replacement for babel and webpack for some of your monorepo libraries but use the CRA based webpack build step for our main app for now.
We build all dependencies of our main app using yarn workspace-tools in parallel (if possible) prior to starting the dev server or building the app.

## 💻 Code Sample
I can reproduce this issue with this minimal repo:
https://github.com/valkum/parcel-segfault-test

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.3.1
| Node             | 16.14.0 and 16.9.1
| npm/Yarn         | yarn 3.1.1
| Operating System | linux 5.16.8-arch1-1

We also see this in our CI with node:16-alpine images.

**Comment by kriszyp** [comment]
Thank you, I will take a look at this!

**Comment by LekoArts** [comment]
This is a duplicate of https://github.com/parcel-bundler/parcel/issues/7702 - can you close it in favor of that and comment there?

**Comment by valkum** [comment]
Closing in favor of #7702 which seems to be a dup of this.
